wp-content/themes/workingnyc/views/search/search-results.twig

```diff
@@ -2,7 +2,7 @@
 
 {% block content %}
 {% if term == ''%}
-<div>
+<div class="hidden tablet:hidden desktop:block">
```

Comment:
You should delete this entire div – it still shows up on desktop

http://localhost:8080/?s=

Reply:
Deleted the entire div